src/core/Tokenizer.js

```diff
@@ -54,10 +54,12 @@ export default class Tokenizer {
     // 2. square bracket quoted string (SQL Server) using ]] to escape
     // 3. double quoted string using "" or \" to escape
     // 4. single quoted string using '' or \' to escape
+    // 5. national character single quoted string using N'' or \' to escape
```

Comment:
I think you forgot to update `Tokenizer` `constructor` `cfg.stringTypes` comment.